### PR TITLE
customized classes and id on BootstrapTable (#235)

### LINKED ISSUE #235: Support className on BootstrapTable
The styling on this version is much better than the first one is much better!

However, I have to wrap every ``BootstrapTable`` in a <div/> so I can target your classes. (using code based classes, not stylesheet ones).

It would be nice if you let me pass className into ``BootstrapTable`` and you add whatever is passed in to the outer div.  This would let me get rid of this extra div.

### PR DESCRIPTION
**Pull request by Chun-MingChen**
fix #235 

This PR aims to support customized `className` and `id` on BootstrapTable.

### CODE REVIEW

**Comment by Chun-MingChen** [comment]
Hi @AllenFang,

For previous customized classes, take [rowClasses](https://github.com/react-bootstrap-table/react-bootstrap-table2/blob/master/docs/README.md#rowClasses) or example,  it was accepted `String` and `Function`. However, callback function might take a couple of params like `row` and `rowIndex` and I thought it might improper and unnecessary for the table. That's the reason why I only support `string` here.

**Comment by Chun-MingChen** [comment]
Besides, should I update the docs as well?

File: packages/react-bootstrap-table2/src/bootstrap-table.js [hunk 1]
```diff
@@ -74,7 +76,7 @@ class BootstrapTable extends PropsBaseResolver(Component) {
 
     return (
       <div className="react-bootstrap-table">
-        <table className={ tableClass }>
+        <table id={ id } className={ tableClass }>
```
Comment (AllenFang) [inline packages/react-bootstrap-table2/src/bootstrap-table.js]: About #235, I think we need to clarify the class/id is on outter div or `table`

**Comment by AllenFang** [comment]
Document should be patched only for `dev` branch, I will update offical docs

File: packages/react-bootstrap-table2/src/bootstrap-table.js (hunk 1 above)
Comment (Chun-MingChen) [reply packages/react-bootstrap-table2/src/bootstrap-table.js]: Sure. Let's discuss further. What I think is that the benefit of customizing `className` or `id` on the table directly is that it not only allows users to pick up whatever they want but also supports `classNames` from `boostrap`. 

For example: [bootstrap4#tables](https://getbootstrap.com/docs/4.0/content/tables/#examples)
> You can also invert the colors—with light text on dark backgrounds—with .table-dark.

```html
<table class="table table-dark">
  { ... }
</table>
```

**Comment by AllenFang** [comment]
Thanks @Chun-MingChen

**Comment by AllenFang** [comment]
Patch docs https://github.com/react-bootstrap-table/react-bootstrap-table2/commit/c11539b9fb1c0ce87f653ea836b6cac375cd3723

**Comment by waynebrantley** [comment]
@AllenFang 
Why create a ``classes`` attribute, instead of using the standard ``className`` attribute and merging it in?
Just makes easier to self discover API.

**Pull request merged**